source/component/tipue/tipuesearch_set.js

```diff
@@ -42,13 +43,15 @@ var tipuesearch_stem = {'words': [
 ]};
 
 
-// Related searches
+// Related
 
-var tipuesearch_related = {'searches': [
-     // Not relevant for PasDoc docs: {'search': 'tipue', 'related': 'Tipue Search'},
-     // Not relevant for PasDoc docs: {'search': 'tipue', 'before': 'Tipue Search', 'related': 'Getting Started'},
-     // Not relevant for PasDoc docs: {'search': 'tipue', 'before': 'Tipue', 'related': 'jQuery'},
-     // Not relevant for PasDoc docs: {'search': 'tipue', 'before': 'Tipue', 'related': 'Blog'}
+var tipuesearch_related = {'Related': [
```

Comment:
As the original comments here say: some of these lines are not relevant for PasDoc (having "related" results that lead to things not interesting to Pascal developers that read/search docs). Please remove these lines.

Reply:
I'm not take a look inside the js, just download the latest version and convert it with the tools in pasdoc

Reply:
You still left 
`{'search': 'tipue', 'related': 'Search', 'include': 1}`
and
`{'search': 'tipue', 'related': 'Support'}`.

If you want to leave them, it may be OK -- but add a comment why :) We want to have here only stuff really interesting to readers of a particular API documentation. These readers are not interested in Tipue, they also didn't develop the Tipue+PasDoc integration, they just used "search" in some API docs they found on the Internet.

Reply:
Added "// Not relevant for PasDoc" comment  in sections in file  tipuesearch_set.js